GsaGH/Parameters/2_Geometry/GsaElement1dParameter.cs

```diff
@@ -32,5 +40,16 @@ public override string InstanceDescription
       this.AddRuntimeError($"Data conversion failed from {data.GetTypeName()} to Element1d");
       return new GsaElement1dGoo(null);
     }
+
+    public bool IsBakeCapable => !m_data.IsEmpty;
```

Comment:
that´s a field and doesn´t belong here ;)

Reply:
moved this - the underscores are from Grasshopper by inheriting stuff from `GH_Goo`